ui/src/components/helpers/lists.js

```diff
@@ -0,0 +1,334 @@
+const accessTypeList = ["Unsigned", "Signed", "Alumni", "Admin", "Stewardship"];
+
+const lifestageList = [
+  "Building Blocks",
+  "Undergraduate",
+  "Postgraduate",
+  "Single Adult",
+  "Covenant",
+];
+
+const ministryTeamList = [
+  "Not Applicable",
+  "Intercessory Prayer Team",
+  "Creatives",
+  "Hospitality",
+  "Band",
+  "Audio/Visual",
+  "Creative Worship",
+  "Multimedia",
+  "Building Blocks",
+];
+
+const lifegroupList = [
```

Comment:
will need to update with new life groups

Reply:
yeah will make a small PR for that in the future